ui/package.json

```diff
@@ -23,20 +23,12 @@
         "watch:nashorn:server": "webpack --config ./__config__/webpack.config.nashorn.dev.js --env.server --watch"
     },
     "dependencies": {
-        "emotion": "^7.3.2",
-        "preact": "^8.2.1",
-        "preact-render-to-string": "^3.6.0",
-        "sass-mq": "^3.3.2",
-        "styletron": "^2.5.7",
-        "styletron-client": "^2.5.7",
-        "styletron-preact": "^2.5.7",
-        "styletron-server": "^2.5.7"
+        "@guardian/guui": "1.2.0"
```

Comment:
what do you think about embedding `ui` more in frontend? make `@guardian/guui` a dep of frontend itself, not `ui`?

Reply:
Yarn usually warns if you attempt to install a module at the root workspace level, suggesting it is not something that Yarn wants you to do. Can you think what the benefit of installing at the root might be?

Thinking about this further, I'm not sure we're using Yarn's workspaces feature to its full potential. Ideally we would also make `static` and `__tasks__` into workspaces, to better delineate the boundaries between the various applications that sit under `frontend`. We could keep the root relatively clean, maybe with dependencies of the git hooks, `dev` scripts and misc `tools`.